cmd/kops/create_cluster.go

```diff
@@ -436,6 +424,33 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		}
 	}
 
+	cloud, err := cloudup.BuildCloud(cluster)
+	if err != nil {
+		return err
+	}
+
+	if c.VPCID != "" {
+		cluster.Spec.NetworkID = c.VPCID
+	}
+
+	if c.NetworkCIDR != "" {
+		cluster.Spec.NetworkCIDR = c.NetworkCIDR
+	}
+
+	if cluster.SharedVPC() && cluster.Spec.NetworkCIDR == "" {
+		vpcInfo, err := cloud.FindVPCInfo(cluster.Spec.NetworkID)
```

Comment:
Interesting.. Wondering your thoughts on calling AWS from the `create_cluster.go` file.. We don't do it for anything else (that I know of) and it seems like there might be a few other places this could live. What are we gaining by doing it here?

I recently walked through the same path here - trying to figure out how to handle egress IDs and it (eventually) went from `populateClusterSpec` -> The Model -> `fi.task` where it ultimately ended up in an `fi.task`.. 

Just curious :)